aks/scfcharts.tf

```diff
@@ -35,6 +65,36 @@ resource "helm_release" "scf" {
     values = [
         "${file("${local.chart_values_file}")}"
     ]
+    # scf-config-values
+    set {
+      name = "env.DOMAIN"
+      value = "${var.cap_domain}"
+    }
+    set {
+      name = "env.UAA_HOST"
+      value = "uaa.${var.cap_domain}"
+    }
+    set_sensitive {
+      name = "secrets.CLUSTER_ADMIN_PASSWORD"
+      value = "${var.admin_password}"
+    }
+    set_sensitive {
+      name = "secrets.UAA_ADMIN_CLIENT_SECRET"
+      value = "${var.root_password}"
+    }
+    # stratos-metrics-values
+    set {
+      name = "kubernetes.apiEndpoint"
+      value = "${var.cap_domain}"
+    }
+    set {
+      name = "nginx.username"
+      value = "${var.username}"
+    }
+    set_sensitive {
+      name = "nginx.password"
+      value = "${var.admin_password}"
+    }
```

Comment:
Same comment as above. Metrics/nginx username password helm params are not used for any chart other than metrics.

Reply:
I downloaded the charts and removed any set values that were not included in the chart values. 29e3717e